test/evaluation.spec.js

```diff
@@ -226,6 +226,8 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(error.message).toContain('JSHandles can be evaluated only in the context they were created');
     });
     it('should simulate a user gesture', async({page, server}) => {
+      await page.evaluate(() => document.body.appendChild(document.createTextNode('test')));
```

Comment:
Nit: you can do these all in one evaluate, save some hops.

Reply:
Good point, done. Await and arrow functions make the code so much more readable that it's easy to forget that there are roundtrips over the protocol on each evaluate invocation :-)